docs/REST API/rest-api.md

```diff
@@ -321,6 +327,31 @@ Retrieves the user data for the authenticated user.
     "gravatar_url": "",
     "sso_enabled": false,
     "teams": []
+  },
+  "available_teams" : [
+    {
+      "id": 1,
+      "name": "Workstations",
+      "description": "Employee workstations"
+    }
+  ],
+  "ui_settings": {
+    "hidden_hosts_table_columns": [
```

Comment:
Q - should these be filtered according to tier and RBAC? The UI will filter them when rendering the columns, so it's not necessary.